packages/client/src/data_commons_client_types.ts

```diff
@@ -32,6 +32,13 @@ export interface BaseGetDataRowsParams {
    * Must be a subset of `variables` param.
    */
   perCapitaVariables?: string[];
+  /**
+   * Delimiter for field header.
+   * Example, if fieldDelimiter = ".", entity value header will be "entity.value"
+   *
+   * Default: "__"
+   */
+  fieldDelimiter?: string;
```

Comment:
Perhaps we should add this to the documentation in the readme?

Reply:
Good call-added.